<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisitWindow</class>
 <widget class="QMainWindow" name="VisitWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Стоматологическая клиника</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/t.png</normaloff>icons/t.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>801</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Визиты</string>
     </attribute>
     <widget class="QTableView" name="tableVisit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>771</width>
        <height>291</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QPushButton" name="addVisit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>231</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="visitDelete">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>10</y>
        <width>241</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="visitUpdate">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>221</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Обновить</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Пациенты</string>
     </attribute>
     <widget class="QTableView" name="tableClient">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>771</width>
        <height>291</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QPushButton" name="clientDelete">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>251</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>221</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clientUpdate">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>221</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Обновить</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Услуги</string>
     </attribute>
     <widget class="QTableView" name="tableService">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>441</width>
        <height>291</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QPushButton" name="serviceDelete">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>251</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="serviceAdd">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>221</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="serviceUpdate">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>221</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Обновить</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="Help">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/vop.png</normaloff>icons/vop.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="choiceot">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать отчет</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ot">
    <property name="geometry">
     <rect>
      <x>444</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cоздать отчет</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
